CONTRIBUTING.rst

```diff
@@ -23,3 +23,34 @@ You can download the CLAs here:
 
 If you are an IBMer, please contact us directly as the contribution process is
 slightly different.
+
+======================
+Development Quickstart
+======================
+
+Clone the repo into a folder, set up a `virtual environment <https://virtualenv.pypa.io/en/latest/>`_, 
+install the requirements:
+
+.. code-block:: bash
+
+    $ git clone git clone git@github.com:cloudant/python-cloudant.git
+    $ cd python-cloudant
+    $ virtualenv .
+    $ ./bin/activate
+    $ pip install -r requirements.txt
+    $ pip install -r test-requirements.txt
+    
+Before running the tests, start CouchDB:
+
+.. code-block:: bash
+    
+    $ couchdb
+
```

Comment:
I would also make mention of setting the ADMIN_PARTY shell environment variable. As in: `export ADMIN_PARTY=true`